rotkehlchen/data_import/importers/cryptocom.py

```diff
@@ -374,13 +399,19 @@ def _import_cryptocom_associated_entries(
             # When we convert multiple assets dust to CRO
             # in one time, it will create multiple debited rows with
             # the same timestamp
+            debited_rows = []
             try:
                 debited_rows = m_row['debited']
                 credited_row = m_row['credited']
             except KeyError as e:
-                self.db.msg_aggregator.add_warning(
-                    f'Failed to get {e!s} event at timestamp {timestamp}.',
-                )
+                for row in debited_rows + [credited_row]:
+                    if row:
+                        self.send_message(
+                            row_index=row[INDEX],
+                            csv_row=row,
+                            msg=f'Failed to get {e!s} event at timestamp {timestamp}.',
+                            is_error=True,
+                        )
                 continue
```

Comment:
Readability. Very hard to understand so many indentation levels with a visual separator

```suggestion
                        )

                continue

```